resident/resident-service/src/main/java/io/mosip/resident/util/EventEnum.java

```diff
@@ -601,7 +601,11 @@ public enum EventEnum {
 	GET_TEMPLATES_EXCEPTION("RES-SER-247", RegistrationConstants.SYSTEM, "get templates failure",
 			"get templates by langCode and templateTypeCode is failed", "RES-SER", "Residence service", "NO_ID",
 			"NO_ID_TYPE", RegistrationConstants.APPLICATIONID, RegistrationConstants.APPLICATIONNAME),
+	INVALID_LANGUAGE_CODE("RES-SER-500", RegistrationConstants.SYSTEM, "Invalid language code",
```

Comment:
please document the error code and message to the API story

Reply:
yes done.